integration_test/multiple_services_test.go

```diff
@@ -116,7 +119,7 @@ func (s *multipleServicesSuite) Test_LargeRequest() {
 func (s *multipleServicesSuite) Test_List() {
 	stdout := itest.TelepresenceOk(s.Context(), "-n", s.AppNamespace(), "list")
 	for i := 0; i < s.ServiceCount(); i++ {
-		s.Regexp(fmt.Sprintf(`%s-%d\s*: ready to engage`, s.Name(), i), stdout)
+		s.Regexp(fmt.Sprintf(`%s-%d\s*: ready to engage|intercept`, s.Name(), i), stdout)
```

Comment:
```suggestion
		s.Regexp(fmt.Sprintf(`%s-%d\s*: ready to (engage|intercept)`, s.Name(), i), stdout)
```
nit, but the capturing (or non-capturing) group is needed here, otherwise the regexp matches a string `intercept`.
In other words, it's an alternative between:
- `%s-%d\s*: ready to engage`
- `intercept`